firmware/quadruna/FSM/src/app/app_mainState.c

```diff
@@ -11,30 +11,18 @@
 #include "app_suspension.h"
 #include "app_loadCell.h"
 #include "app_shdnLoop.h"
-#include "app_heartbeatMonitor.h"
 
 void mainStateRunOnTick100Hz(void)
 {
+    app_canTx_FSM_Heartbeat_set(true);
```

Comment:
I'm a bit confused what's going on here. Do any boards consume the FSM heartbeat? if not why not remove the messages altogether?

Reply:
Yes, the VC does for accelerator pedals and brakes.